docs/data-modeling/metadata-editing.md

```diff
@@ -144,14 +144,13 @@ To change a column's [filter widget](../dashboards/filters.md):
 
 #### Filter widget options
 
-- **Search box**: Enter a search term and Metabase will display checkboxes for column values that match the search.
-- **A list of all values**: dropdown menu with checkboxes for all column values.
-- **Plain input box**: Enter a search term and Metabase will make autocomplete suggestions for the search (no checkboxes).
+The default behavior for the `Is` filter for the field.
 
-#### Default filters
+- **Search box**: Display a search box and suggest autocompletions for values in that column that match the search term(s).
+- **A list of all values**: Display a search box, as well as a list of checkboxes for values in a dropdown menu that people can select as search terms.
+- **Plain input box**: Display a search box, but do NOT suggest autocompletions.
 
-- Columns with more than 100 unique values will default to a plain input box filter.
-- Columns with fewer values will display a search box filter.
```

Comment:
Why are we removing this part?

Reply:
The limit is no longer 100. Can you confirm that the limit is 1000?

Reply:
I'll try to find it.

What about the second line that claimed that "Columns with fewer values will display a search box filter."?

Reply:
That line also depends on the limit.